CMakeLists.txt

```diff
@@ -73,7 +73,7 @@ if (NOT VARA_FEATURE_IN_TREE)
     find_package(LLVM ${LLVM_REQUESTED_VERSION} REQUIRED CONFIG)
   else()
     message(STATUS "Not set")
-    find_package(LLVM 10 REQUIRED CONFIG)
+    find_package(LLVM 9 REQUIRED CONFIG)
```

Comment:
We require and support by default LLVM 10 if someone wants to build an older version, cmake offers a extra flag to allow older LLVM version. The readme explains how to build with older versions, but we do not guarantee that everything will work.